subprojects/execution/src/main/java/org/gradle/internal/execution/steps/ValidateStep.java

```diff
@@ -70,6 +75,7 @@ public ValidateStep(
     public R execute(UnitOfWork work, BeforeExecutionContext context) {
         WorkValidationContext validationContext = context.getValidationContext();
         work.validate(validationContext);
+        context.getBeforeExecutionState().ifPresent(beforeExecutionState -> validateImplementations(work, beforeExecutionState, validationContext));
```

Comment:
Split this to two lines.
```suggestion
        context.getBeforeExecutionState()
            .ifPresent(beforeExecutionState -> validateImplementations(work, beforeExecutionState, validationContext));
```

Reply:
Done.